apps/admin-gui/src/assets/i18n/en.json

```
@@ -629,7 +629,8 @@
         "DELETE_APPLICATION_DESCRIPTION": "Application with following Id will be deleted.",
         "DELETE_MESSAGE": "Application was deleted",
         "APPROVE_MESSAGE": "Application was approved",
-        "VERIFY_MESSAGE": "Application was verified"
+        "VERIFY_MESSAGE": "Application was verified",
+        "AUTO_APPROVE_FAILURE": "Application auto-approve failed"
```

I would slightly change the message.
```suggestion
        "AUTO_APPROVE_FAILURE": "Automatic approval of application failed"
```